site/pages/home/contact-us-slice/index.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/no-set-state */
+import React, { Component } from 'react';
+import Form from './form/';
+import Success from './success/';
+
+class ContactUs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      errors: {},
+      success: false,
+      fatalError: false,
+    };
+    this.submitForm = this.submitForm.bind(this);
+    this.onClose = this.onClose.bind(this);
+  }
+
+  onClose() {
+    this.setState({
+      success: false,
+    });
+  }
+
+  submitForm(formData) {
+    const formDataJSON = JSON.stringify(formData);
+    fetch('https://xmy0g2tvu0.execute-api.eu-west-1.amazonaws.com/dev/contact-us',
+      {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        mode: 'cors',
+        body: formDataJSON,
+      })
+      .then(response => response.json())
+      .then(json => {
+        if (json.errorMessage) {
```

Comment:
errors return a `200` from the endpoint. Checked with @asavin that this is acceptable behaviour.
Conclusion - yes as it is a private api, be sure to document it